Main_Solution/main_solution_test.rb

```diff
@@ -135,10 +146,13 @@ def test_conditional_watch_issue
     end
 
     @wait.until{is_issue_watched?}
-    assert(is_issue_watched?)
+    expect(is_issue_watched?).to be
```

Comment:
doesn't it look like strange assertion?
the line 149 will never fail :)
try to find another way